src/Infrastructure/Repositories/EmployeeRepository.cs

```diff
@@ -210,17 +209,19 @@ private async Task AddCertificationUncommitted(List<Certification> certification
                 await _db.AddAsync(certificationEntity);
                 continue;
             }
-
+            else{
             certificationEntity.Title = certification.Title;
             certificationEntity.Description = certification.Description;
             certificationEntity.ExpiryDate = certification.ExpiryDate;
             certificationEntity.IssuedMonth = certification.IssuedMonth;
             certificationEntity.IssuedYear = certification.IssuedYear;
             certificationEntity.LastSynced = DateTime.Now;
+            }
+            await _db.SaveChangesAsync();
         }

```

Comment:
Burde det være en `continue;` inne i `else` blokken? Og skal `await _db.SaveChangesAsync();` være inne i loopen?

Reply:
Indentation på else blokken burde fikses, samme lenger nede :D

Reply:
Vi kan prøve med alle rader for tabellen, da sparer vi nok tid ja. Skal ikke være continue.